src/extensions/group-by-v2/bootstrap-table-group-by.js

```diff
@@ -190,9 +190,16 @@ BootstrapTable.prototype.initBody = function (...args) {
     if (this.options.groupByToggle) {
       this.$container.off('click', '.groupBy')
         .on('click', '.groupBy', function () {
+          const groupIndex = $(this).closest('tr').data('group-index')
```

Comment:
Now that we're here could we make a const variable for `$(this)` something like `$this` and then replace it in this click function?